contracts/contracts/interfaces/IWETH9.sol

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: MIT
```

Comment:
nit: this could have `deposit` and `withdrawal` functions and events and inherit other stuff from IERC20.sol I think

Reply:
Could indeed. I figured there was no harm in using an export from the actual WETH contract.